data/unified-test-format/valid-pass/initialCollectionData-collectionOptions.yml

```diff
@@ -0,0 +1,39 @@
+description: initialCollectionData-collectionOptions
+
+schemaVersion: '1.5'
+
+createEntities:
+  - client:
+      id: &client0 client0
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name database0
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name coll0
+
+initialData:
+  - collectionName: *collection0Name
+    databaseName: *database0Name
+    collectionOptions:
+      capped: true
+      size: &cappedSize 512
+    documents:
+      - { _id: 1, x: 11 }
+
+tests:
+  - description: collection is created with the correct options
+    operations:
+      # Execute a collStats command to ensure the collection was created with the correct options.
+      - name: runCommand
+        object: *database0
+        arguments:
+          commandName: collStats
+          command:
+            collStats: *collection0Name
+            scale: 1
+        expectResult:
+          capped: true
+          maxSize: *cappedSize
```

Comment:
This test doesn't pass on server version 2.6 and 3.0. Discussion in underway [in the spec PR](https://github.com/mongodb/specifications/pull/959/files#r853270709) to add `minServerVersion: "3.6"`. I'll add it in this PR for now and only merge if my suggestion is taken in the spec PR.